_pytest/vendored_packages/attr/__init__.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
@jaraco vendoring atts was suggested as attrs is tested using pytest - and well the only supported way in python to import multiple versions of a package at the same time is to put them under a new name to begin with - even setuptools didnt solve that as multi version install never supported multi version import